requirements.txt

```diff
@@ -1,3 +1,4 @@
+alabaster<0.8,>=0.7
```

Comment:
I don't feel comfortable maintaining this. We had the same problem with `urllib3` in the past. I don't want to do the same mistake again.

Reply:
> None of them. Look at the patch. [2a7bccb](https://github.com/funilrys/PyFunceble/commit/2a7bccbf64813db811505bac53e39ae630cf26bc)
> 
> In other words: I handle both cases.

Saw that one after committing :smiley: